### PR TITLE
New add-on applicationDictionary

### PR DESCRIPTION
**Pull request by ruifontes**
### Release information
- Name: Application Dictionary
- Author: Rui Fontes <rui.fontes@tiflotecnia.com> and Ricardo Leonarczyk <ricardo.leonarczyk95@gmail.com>"
- Repo: https://github.com/ruifontes/applicationDictionary
- Version: 2022.03
- Update channel: stable
- NVDA compatibility: 2022.1 and beyond): 

### Changelog (mention changes in separate lines starting with dash space)
- First version submitted

## Usage
This simple add-on makes possible to create application-specific dictionaries for NVDA.
To use the add-on, just go to the application for wich you want a dictionary for, and press NVDA+shift+p.
The normal NVDA dictionary dialog will open for you to add entries valid only for the active application (the application name should appear in the dialog title).
The default add-on shortcut can be changed through NVDA's input gestures dialog: under configuration node, expand the node that says "Shows the application-specific dictionary dialog".

## Command
NVDA+Shift+p

## Automatic update
This add-on includes an automatic update feature.
The check for a new version will be executed everytime NVDA is loaded.
If you do not want this, go to NVDA, Preferences, Options and in the add-on category uncheck the check box.

### CODE REVIEW

File: get.php [hunk 1]
```diff
@@ -2,6 +2,8 @@
 $addons = array(
 	"access8math" => "https://github.com/tsengwoody/Access8Math/releases/download/v3.2/Access8Math-3.2.nvda-addon",
 	"addonshelp" => "https://github.com/ruifontes/addonsHelp/releases/download/2022.03/addonsHelp-2022.03.nvda-addon",
+	"appdict" => "https://github.com/ruifontes/applicationDictionary-/releases/download/2022.03/applicationDictionary-2022.03.nvda-addon
+",
```
Comment (CyrilleB79) [inline get.php:6]: Please remove extra newline character here.

**Review by CyrilleB79 — CHANGES_REQUESTED** [review]
My comments here:
Since this add-on is only compatible with NVDA 2022.1, this PR should not be merged before NVDA 2022.1 stable is released.
If it is merged earlier, we would offer on the community website an add-on incompatible with last NVDA stable version (2021.3.5).
Same issue as https://github.com/nvaccess/addonFiles/pull/278#issuecomment-1078424971.

**Comment by CyrilleB79** [comment]
Of course, as explained and decided in #278, this PR can be merged immediately (without waiting 2022.1 stable) if you provide another version of the add-on whose compatibility range contains NVDA 2021.3.5 (current stable version). That's probably possible just changing the compatibility flags since add-ons written for 2022.1 are usually also compatible with 2021.3.

If you provide another version of your add-on, it's recommended that you set a new version number (e.g. 2022.03.1) to avoid confusion.I seem to remember that this requirement was not respected for another recently released add-on, leading to confusion while testing.
Thanks for your understanding.

**Comment by nvdaes** [comment]
Hello: I wanted to report an issue in the mentioned repo but I cannot access it. 🤔

The issue is that, when reloading plugins pressing NVDA+control+f3, the menu item of the add-on (Application dictionary) appears multiple times under Preferences, Speech Dictionaries menu.

**Comment by ruifontes** [comment]
Closed.
A new PR will be made with all problems solved.

**Pull request closed without merging**